test/replays/132/replays.test.ts

```diff
@@ -1,11 +1,20 @@
+import { jest, it } from "@jest/globals";
+import console from "node:console";
 import W3GReplay from "../../../src/";
-import path from "path";
+import path from "node:path";
 import {
   GameDataBlock,
   TimeslotBlock,
 } from "../../../src/parsers/GameDataParser";
+import { spyConsoleError } from "../../__utils__/spyConsole";
```

Comment:
I think extracting this into a utility function is a little bit of overkill. I would rather just use it on demand if needed in a test file because people are used to the jest testing utility on a first glance, the spy / mock is not complex and we can set it up in the global scope of the test file if its needed for multiple test cases, like this:

```
const consoleLogSpy = jest.spyOn(console, "log")
const consoleErrorSpy = jest.spyOn(console, "error")

it('...', () => {
  //assert consoleLogSpy not called
  //assert consoleErrorSpy
})
```

Also, please add an assertion that console.log was not called any more during a parser run so that we detect if someone accidentally adds a log to console.log in the future.